provider/datagen/src/transform/cldr/datetime/mod.rs

```diff
@@ -320,6 +324,26 @@ impl_data_provider!(
     symbols::convert_dates,
     calendared = "indian"
 );
+impl_data_provider!(
+    IslamicObservationalDateSymbolsV1Marker,
+    symbols::convert_dates,
+    calendared = "islamic"
+);
+impl_data_provider!(
+    IslamicCivilDateSymbolsV1Marker,
+    symbols::convert_dates,
+    calendared = "islamicc"
+);
+impl_data_provider!(
+    UmmAlQuraDateSymbolsV1Marker,
+    symbols::convert_dates,
+    calendared = "umalqura"
```

Comment:
I sat down with Shane and we concluded these are best for now because I was having problems with dashes. For example in the format "islamic-calendar" it would only download the data for the first word "islamic" (observational)

Reply:
feels like a datagen or locale parsing bug, we should have a followup then

Reply:
sounds good, we can discuss it tomorrow

Reply:
`value!()` only works with a single subtag, and there may be some code that tries to turn these into tinystrs that don't have enough capacity. Shortening to a single subtag is temporary. I think it should be mostly a non-issue moving forward because I want to see us consolidate this down to a single key anyway, so resolution should only happen in AnyCalendar and not anywhere else.